### ISSUE TITLE
Changes to Deckplan wanted bei Ulf

### ISSUE THREAD

**Issue by ue71603** [NeTEx-CEN/NeTEx#702]
xsd/netex_framework/netex_reusableComponents/netex_spotEquipment_version.xsd
I lack (optional) attributes/elements indicating the relative position in meters in the X and Y direction compared with the enclosing PASSENGER SPACE left bottom corner when the PASSENGER SPACE is viewed straight from above assuming its normal direction of travel is straight to the left. X is then the distance to the right and Y is the distance upwards. (I note that the Z direction is already handled with the HeightFromFloor.)

xsd/netex_framework/netex_reusableComponents/netex_seatingPlan_version.xsd
I lack (optional) attributes/elements indicating the relative position in meters in the X and Y direction compared with the enclosing DECKs left bottom corner when the DECK is viewed straight from above assuming its normal direction of travel is straight to the left. X is then the distance to the right and Y is the distance upwards.

xsd/netex_framework/netex_reusableComponents/netex_deckPlan_version.xsd
As a DeckEntrance can be referred to from two DeckSpaces I think it should be possible to (also) define them (but not refer them) at DECK level (i.e. in DeckGroup) and not only inside a certain DeckSpace. That means a profile could then require them to be defined at DECK level and forbid that they are defined inside a DeckSpace. Thus always using references (e.g. DeckEntranceRef) inside DeskSpaces.

**Comment by skinkie** [comment]
Lift of Enumeration Aisle, Window change back to IsWindow, IsAisle boolean.

**Comment by nick-knowles** [comment]
Current TRansmodel suggestion is 
   (a) Revise definition of ZONE to be just an  abstract 2d space of any coordinate system
   (b) Rename EQUIPABLE SPACE to ONBOARD SPACE (sp that PASSENGER SPACE, SPOT etc have coordinates)
   (c)  Make both  DECK and ONBOARD SPACE as subclasses of ZONE (ie 2d space) 
   (d) Use DECK ZONE as boynding box for ONBOARD SPACE components
    
  (c)  Rename TRAIN STOP ASSIGNMENT to BOARDING POSITION STOP ASSIGNMENT (Will have to keep TrainStopAssignment in netex as deprecated alias)
  Q,  DECKSs can overlap What happens if an ONBOARD SPACE is on more than one deck eack with a different coordinate bounding box  (Poss use  ZONE PROJECTION and COMPLEX FEATURES?)
   Q2 SHould we also add a height to ONBOARD SPACE so that all spaces have heights
    Q4 In TRANSMODEL it makes sense to merge DECK ASSIGNMENT with BOARDING POSITION ASSIGNMENT (formerly TRAIN STOP ASSIGNMENT) and DECK ENTRANCE ASSIGNMENT into a single package (Moving PASSENGER STOP ASSIGNMENT to Part3 since it now depends on SERVICE JOURNEY)  . SHoudl we do the same in NeTEx?  
![image](https://github.com/NeTEx-CEN/NeTEx/assets/1826507/7ee8c86b-de2d-48ee-aa0e-1d56e474cb2b)

**Comment by nick-knowles** [comment]
You are quite right,  there is a gap here as to how to specify the precise location of equipment within an ONBOARD SPACE 


1.  DECK SPACEs, PASSENGER SPACE, LOCATABLE SPOT etc are all descendants of ONBOARD SPACE  which in turn is a specialisation of ZONE , so may be given a bounding polygon and a centroid in coordinates relative to the DECK. 

<img width="1179" height="841" alt="Image" src="https://github.com/user-attachments/assets/17642200-5fe9-4233-bf9e-7fe6f6483744" />



-------------------
2. EQUIPMENT can currently be associated with various deck components, indicating that it is colocated with the component, but precise relative  coordinates within the deck component cannot be given

<img width="1174" height="877" alt="Image" src="https://github.com/user-attachments/assets/46ea8b66-b861-4926-933d-3173a8f85d35" />



----------------
3. In the analogous SITE mode, Equipment can be exactly  located with an  EQUIPMENT PLACE and EQUIPMENT POSITION elements.   

<img width="794" height="754" alt="Image" src="https://github.com/user-attachments/assets/65e27458-268c-467a-8b3a-b191028dd304" />

-------------------

Solution
We could add  a link between ONBOARD SPACE and EQUIPMENT PLACE to allow a precise statement of position in relative coordinates (rQ. elative to the DECK or the parent space?). This is preferable to placing a coordinate on EQUIPMENT (which would mix concerns)
